frontends/verific/verific.cc

```diff
@@ -441,7 +441,11 @@ bool VerificImporter::import_netlist_instance_gates(Instance *inst, RTLIL::IdStr
 		return true;
 	}
 
-	// FIXME: PRIM_DLATCH
+	if (inst->Type() == PRIM_DLATCH)
```

Comment:
This block completely ignores async cond and val, so in end result S inputs do not affect latch which is wrong. When same code as in lines 575-586 is used it gives expected result ( also it matches results as when I put "db_infer_set_reset_registers" back to 1)

Reply:
Thanks! TFW You realize you spent 30 minutes staring at the wrong code last night..  :D  
Fixed in 90b440f.